Sources/Datadog/RUM/RUMMonitor/Scopes/RUMResourceScope.swift

```diff
@@ -233,6 +233,7 @@ internal class RUMResourceScope: RUMScope {
                     url: resourceURL
                 ),
                 source: command.errorSource.toRUMDataFormat,
+                sourceType: command.errorSourceType,
```

Comment:
This is not covered in `RUMResourceScopeTests.swift` - if we put an arbitrary value here, no test will trigger.